scripts/202306xx_populate_paymenttype_default/add_fixed_payment_type_default.js

```diff
@@ -4,31 +4,87 @@
 Add PaymentType.Fixed to all forms with existing payments_field object
 */
 
-// Count number of forms that contains payments_field property
-db.getCollection('forms')
-  .find({ payments_field: { $exists: true } })
-  .count()
+// Batch 1 2022
+const queriesModifiedIn2022 = {
+  payments_field: { $exists: true },
+  lastModified: { $gte: new ISODate('2022-01-01T00:00:00.000+0000') },
```

Comment:
/nit should we combine the two [checks](https://www.mongodb.com/docs/manual/reference/method/db.collection.find/?_ga=2.237881088.1020962104.1688274645-959180788.1669692925#query-for-ranges)?

`lastModified: { $gte: new ISODate(), $lt: new ISODate() }`

Reply:
Oh, thanks for the tip! Yes that looks terser.